dotnet/src/Functions/Functions.OpenAPI/OpenAI/OpenAIAuthenticationConfig.cs

```diff
@@ -3,7 +3,7 @@
 using System;
 using System.Collections.Generic;
 using System.Runtime.Serialization;
-using System.Text.Json.Serialization;
+using Newtonsoft.Json;
```

Comment:
@dmytrostruk I switched back to using `Newtonsoft.Json` since serialization of the enum values doesn't seem to work well with the native `System.Text.Json` library, as shown [here](https://dotnetfiddle.net/g2lIFA). maybe I'm doing something wrong?